Dockerfile

```diff
@@ -18,6 +18,9 @@ RUN apk add --no-cache bash && ./build/sbt -info clean package
 # Small runtime image
 FROM alpine:${ALPINE_VERSION} as runtime
 
+# Resolve missing libc package
+RUN apk update && apk add --no-cache libc6-compat
```

Comment:
@tnightengale  do you have any concerns with updating an older version of `libc6-compat`?  Per the issue original issue, I believe this was an issue due to running the docker on a Windows environment.  I don't believe there is an issue with current *nix environments but figured I would check with you on this?